CliWrap/Builders/ArgumentsBuilder.cs

```diff
@@ -2,35 +2,29 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using System.Security;
 using System.Text;
 
 namespace CliWrap.Builders;
 
 /// <summary>
 /// Builder that helps generate well-formed arguments string.
 /// </summary>
-public partial class ArgumentsBuilder
+public class ArgumentsBuilder
 {
     private static readonly IFormatProvider DefaultFormatProvider = CultureInfo.InvariantCulture;
 
-    private readonly StringBuilder _buffer = new();
+    private readonly SecureString _buffer = new();
```

Comment:
As far as I know, `SecureString` doesn't do anything on .NET anymore, so this is pointless. https://github.com/dotnet/platform-compat/blob/master/docs/DE0001.md

![image](https://user-images.githubusercontent.com/1935960/164229735-5021665e-c161-4b16-8cb4-8891f6974565.png)